packages/replay/metadata/index.ts

```diff
@@ -24,9 +34,9 @@ function sanitize(metadata, opts = {}) {
     if (value === null || key.startsWith("x-")) {
       // passthrough null or userspace types
       updated[key] = value;
-    } else if (ALLOWED_KEYS.includes(key)) {
+    } else if (isAllowedKey(key)) {
       // validate known types
-      Object.assign(updated, handlers[key](metadata));
+      Object.assign(updated, handlers[key](metadata as any));
```

Comment:
:speech_balloon: Probably a better way to do it here, but this is not safety-critical code, so YOLO

Reply:
Agreed. I spent a little time trying to figure out the right type but couldn't and I bailed on it.